src/azure-cli/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -546,7 +546,7 @@
 examples:
   - name: Restart 2 nodes.
     text: >
-        az sf managed-node-type node restart -g testRG -c testCluster -n snt --node-name snt_0 snt_1
+        az sf managed-node-type node restart -g testRG -c testCluster -n snt --node-name snt_0 snt_1 --update-type ByUpgradeDomain
```

Comment:
I suggest to keep this example as is and add a new one. And in the summary of the new one clarify that it will restart following an Upgrade Domain walk

Reply:
for the new example remove the --node-name. That would be the most typical used cased, to restart the whole node type ud by ud

Reply:
that didn't work when i ran the test